Trading/Exchange/binance/binance.py

```diff
@@ -0,0 +1,178 @@
+                {"symbol": self.get_exchange_pair(symbol),
+                 "limit": limit}))
+        ]
+
+    async def set_symbol_leverage(self, symbol: str, leverage: int):
+        await self.client.fapiPrivate_post_leverage(
+            {"symbol": self.get_exchange_pair(symbol),
+             "leverage": leverage})
+
+    async def set_symbol_margin_type(self, symbol: str, isolated: bool):
+        await self.client.fapiPrivate_post_marginType(
+            {"symbol": self.get_exchange_pair(symbol),
+             "marginType": self.BINANCE_MARGIN_TYPE_ISOLATED if isolated else self.BINANCE_MARGIN_TYPE_CROSSED})
+
+    def cleanup_position_dict(self, position_dict, position_status):
+        try:
+            # If exchange has not position id -> global position foreach symbol
+            set_exchange_value_to_default(position_dict, ExchangeConstantsPositionColumns.ID.value,
+                                          position_dict[ExchangeConstantsPositionColumns.SYMBOL.value])
+
+            set_exchange_value_if_necessary(position_dict, ExchangeConstantsPositionColumns.QUANTITY.value,
+                                            self.BINANCE_FUTURE_QUANTITY)
+
+            set_exchange_value_if_necessary(position_dict, ExchangeConstantsPositionColumns.MARK_PRICE.value,
+                                            self.BINANCE_MARK_PRICE)
+
+            set_exchange_value_if_necessary(position_dict, ExchangeConstantsPositionColumns.ENTRY_PRICE.value,
+                                            self.BINANCE_ENTRY_PRICE)
+
+            set_exchange_value_if_necessary(position_dict, ExchangeConstantsPositionColumns.LIQUIDATION_PRICE.value,
+                                            self.BINANCE_FUTURE_LIQUIDATION_PRICE)
+
+            set_exchange_value_if_necessary(position_dict, ExchangeConstantsPositionColumns.UNREALISED_PNL.value,
+                                            self.BINANCE_FUTURE_UNREALIZED_PNL)
+
+            # calculated keys
+            set_exchange_value_to_default(position_dict, ExchangeConstantsPositionColumns.VALUE.value,
+                                          calculate_position_value(
+                                              float(position_dict[ExchangeConstantsPositionColumns.QUANTITY.value]),
+                                              float(position_dict[ExchangeConstantsPositionColumns.MARK_PRICE.value])
+                                          ))
+
+            set_exchange_value_to_default(position_dict, ExchangeConstantsPositionColumns.STATUS.value, position_status)
+
+            # TODO
+            set_exchange_value_to_default(position_dict, ExchangeConstantsPositionColumns.REALISED_PNL.value, 0)
+            set_exchange_value_to_default(position_dict, ExchangeConstantsPositionColumns.MARGIN.value, 0)
+            set_exchange_value_to_default(position_dict, ExchangeConstantsPositionColumns.TIMESTAMP.value, time.time())
```

Comment:
From @GuillaumeDSM in https://github.com/Drakkar-Software/OctoBot-Tentacles/pull/147#discussion_r394693277
> If it's right after for other exchanges then ok but otherwise if we have ms timestamps averywhere when using binance except here, I think it can complicate stuff, what do you think about it ?